arangodb-net-standard.Test/ViewApi/ViewApiClientTest.cs

```diff
@@ -129,10 +125,9 @@ public async Task PutRenameViewAsync_ShouldSucceed()
                 testName,
                 new PutRenameViewBody()
                 {
-                     Name = newName
+                    Name = newName
                 });
 
-            Assert.Equal(HttpStatusCode.OK, res.Code);
```

Comment:
This line was causing a test failure.

ArangoDB behaviour appears to have changed in some release >3.10.0 and the `Code` property is `0` instead of the expected `200`.